src/Database/Esqueleto/Internal/Internal.hs

```diff
@@ -1405,26 +1426,105 @@ data Insertion a
 -- are no guarantees that they will behave the same.
 --
 -- @since 2.2.7
-data LockingKind
-    = ForUpdate
+data LockingKind where
```

Comment:
Can probably more easily write the deriving clause with non-GADT syntax

Reply:
Ah this was an artifact from the previous implementation, fixed thanks!